### DISCUSSION TITLE
Documentation Question

### DISCUSSION THREAD

**Discussion by nerdburt** [open-watcom/open-watcom-v2#792] [Q&A]
Good day, I was wondering where I could direct a documentation fix request to?

I found a minor error on page 223 / 318 of the pguide.PDF file.   Section 29.1: The Sample GUI Application, has a function call to WinMain(), and the parameter types are incorrect, when I tried to compile for WinNT model.

The line has 
int PASCAL WinMain( HANDLE hInstance, HANDLE hPrevInst,
LPSTR lpCmdLine, int nCmdShow )

but I believe the line should state
int PASCAL WinMain( **HINSTANCE** hInstance, **HINSTANCE** hPrevInst,
LPSTR lpCmdLine, int nCmdShow )

Tested correctly, but another set of eyes always welcomed.

**Comment by jmalak** [comment]
It is difference between standard and strict mode for 16-bit Windows against WIN32 form.
OW documentation is written in Watcom General Markup Language (.gml file extension).
It is located in docs/doc subtree.
I will correct it, because it is small change. If you will plan do more changes then it is better to familiare with OW documentation organization. If you raise [issue](https://github.com/open-watcom/open-watcom-v2/issues) I can explain some details.
It is submitted as 45e21647 commit
